src/main/java/net/sf/jabref/external/AutoSetExternalFileForEntries.java

```
@@ -176,7 +175,6 @@ public void run() {
                             // Cancel
                             break mainLoop;
                         }
-                        brokenLinks++;
```

Shouldn't "brokenLinks" be output somehow than just be silently ignored?